### PR TITLE
Add a footer

### PR DESCRIPTION
**Pull request by igor-romasiuk**

### CODE REVIEW

File: src/components/Footer/Footer.module.scss [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  gap: 32px;
+
+  @include respond-to(tablet) {
+    justify-content: space-between;
+    flex-direction: row;
+    padding: 32px;
+  }
+
+  @include respond-to(desktop) {
+    padding: 32px 152px;
```
Comment (coldshein) [inline src/components/Footer/Footer.module.scss:22]: we don't need the inline paddings, just add <Container/> in footer

File: src/components/Footer/Footer.module.scss [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  display: flex;
+  gap: 16px;
+  align-items: center;
+
+  .backToTopTitle {
+    font-size: 12px;
+    color: #7A7D8C;
+  }
+
+  .backToTopBtn {
```
Comment (coldshein) [inline src/components/Footer/Footer.module.scss:71]: make the snake case

File: src/components/Footer/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+                <nav className={styles.navigation}>
+                    <a href="#">
+                    Github
+                    </a>
+                    <a href="#">Contacts</a>
+                    <a href="#">Rights</a>
+                </nav>
+                <div className={styles.backToTop}>
+                    <p className={styles.backToTopTitle}>Back to top</p>
+                    <button className={styles.backToTopBtn} onClick={handleBackToTop}>↑</button>
```
Comment (coldshein) [inline src/components/Footer/Footer.tsx:26]: also snake case

**Review by coldshein — CHANGES_REQUESTED** [review]
Great, fix some moments that I've written in the comments

**Comment by igor-romasiuk** [comment]
Hi! I have made the snake case but I have problem with inline paddings in footer

**Pull request closed without merging**